repository/src/main/java/org/alfresco/repo/security/authentication/identityservice/IdentityServiceFacade.java

```diff
@@ -52,6 +55,14 @@ public interface IdentityServiceFacade
      */
     DecodedAccessToken decodeToken(String token) throws TokenDecodingException;
 
+    /**
+     * Gets claims about the authenticated user,
+     * such as name and email address, are retrieved from the UserInfo endpoint of the OpenID provider.
+     * @param token {@link String} with encoded access token value.
+     * @return {@link UserInfo} containing user claims.
+     */
+    Optional<UserInfo> getUserInfo(String token);

```

Comment:
Would be good to not depend on the nimbus classes here.

Reply:
Changed to `org.alfresco.repo.security.authentication.identityservice.OIDCUserInfo`.